pkg/universe.dagger.io/x/david@rawkode.dev/kubernetes/kubectl.cue

```diff
@@ -0,0 +1,43 @@
+package kubectl
+
+import (
+	"dagger.io/dagger"
+	"universe.dagger.io/docker"
+)
+
+#Apply: {
+	manifests:        dagger.#FS
+	kubeconfigSecret: dagger.#Secret
+
+	// Kubernetes version
+	version: string | *"latest"
+
+	_pull_image: docker.#Pull & {
+		source: "bitnami/kubectl:\(version)"
+	}
+
+	apply: docker.#Run & {
+		input:   *_pull_image.output | docker.#Image
+		workdir: "/work"
+		entrypoint: ["kubectl"]
+		command: {
+			name: "apply"
+			args: [
+				"--kubeconfig",
+				"/kubeconfig",
```

Comment:
You can eliminate these lines if you mount directly into `/.kube/config`

https://github.com/bitnami/bitnami-docker-kubectl/blob/2d1e0a2b377107a19b27b95bd851df41c44a2bfc/1.23/debian-10/Dockerfile#L19

Reply:
Another great improvement for later, when someone is up for owning the official `kubernetes` package. Last time that I checked, we had two experimental kubernetes packages (we may have more by now). Merging all the learnings into a single official `kubernetes` package sounds like a great idea to me!